File: java/Aksesoris.java
public class Aksesoris extends Petshop {
    private String jenis;
    private String bahan;
    private String warna;

    // Konstruktor default
    public Aksesoris() {
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }

    // Getter dan Setter
    public String getJenis() {
        return jenis;
    }

    public void setJenis(String jenis) {
        this.jenis = jenis;
    }

    public String getBahan() {
        return bahan;
    }

    public void setBahan(String bahan) {
        this.bahan = bahan;
    }

    public String getWarna() {
        return warna;
    }

    public void setWarna(String warna) {
        this.warna = warna;
    }
}

File: python/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    __jenis = ""
    __bahan = ""
    __warna = ""
    
    def __init__(self, jenis = "", bahan = "", warna = ""):
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna
        
    def get_jenis(self): return self.__jenis
    def set_jenis(self, jenis): self.__jenis = jenis
        
    def get_bahan(self): return self.__bahan
    def set_bahan(self, bahan): self.__bahan = bahan
        
    def get_warna(self): return self.__warna
    def set_warna(self, warna): self.__warna = warna